fhservices.cfg.j2

```diff
@@ -82,11 +88,21 @@ define service {
 {% endfor %}
 
 define service {
+      service_description keycloak:Health
+      hosts {{ rhmap_router_dns }}
```

Comment:
@mikenairn Probably best to get https://github.com/feedhenry/nagios-container/pull/20 merged first in this case. Only a very minor change in this PR then to make it work. I can look at that PR now.